executor/executor.go

```diff
@@ -1156,6 +1163,70 @@ func (e *LimitExec) adjustRequiredRows(chk *chunk.Chunk) *chunk.Chunk {
 	return chk.SetRequiredRows(mathutil.Min(limitTotal, limitRequired), e.maxChunkSize)
 }
 
+func minUint64(a, b uint64) uint64 {
```

Comment:
You can use `mathutil.MinUint64` instead.

Reply:
Nice, thanks!